src/plugins/kibana/public/visualize/editor/agg_group.html

```diff
@@ -5,7 +5,7 @@
 
   <div class="vis-editor-agg-group" ng-class="groupName">
     <!-- wrapper needed for nesting-indicator -->
-    <div ng-repeat="agg in group" class="vis-editor-agg-wrapper">
+    <div class="vis-editor-agg-wrapper" ng-repeat="agg in group">
```

Comment:
Is there a reason for this change? In general we try to keep the angular attributes first, though it's not formalized in the style guide or even used in this file :balloon: